apps/dashboard/app/javascript/xdmod/jobs.js

```diff
@@ -132,12 +153,25 @@ function tableRows(context, helpers) {
     td3.innerText = helpers.date(job);
 
     // <td>{{cpu_label cpu_user}}</td>
-    const td4 = document.createElement('td');
-    td4.innerHTML = helpers.cpu_label(job.cpu_user);
+    // Not used with new analytics data
+    // const td4 = document.createElement('td');
+    // td4.innerHTML = helpers.efficiency_label(job.cpu_user);
 
-    tr.append(td1, td2, td3, td4);
+    tr.append(td0, td1, td2, td3);
 
     rows.push(tr);
+
+    // Add job analytics placeholder
+    const analyticsRow = document.createElement('tr');
```

Comment:
I think there's something odd about creating a new table row here. Screen readers will get tripped up when row N (a row they've looked at) is now suddenly different because it was the initial row, but after this expansion it's now this new data.

Shouldn't it put all this stuff in the original `td` and just use some styling to achieve the same look & feel?

Reply:
I can improve the accessibility of the hidden rows with `aria-hidden` and `aria-expanded` and `aria-controls`. Wouldn't that address the screen readers issue?

I am not sure how feasible is to add content to the original td that contains the +/- icon and make it break out of the existing table structure, push the other rows down, and appear in the same way. Or even to have a hidden td and do the same. Will need to investigate.

It would be more feasible to appear as a small pop up / tooltip, but I think that would be less user friendly.

Reply:
> I can improve the accessibility of the hidden rows with aria-hidden and aria-expanded and aria-controls. Wouldn't that address the screen readers issue?

No it's the fact that it's a `tr` that's being created.  Imagine this - you saw row 2 which the screen reader read out as `row 2`. Then you expand this thing in row 1 and now all the sudden row 2 is this new information, not what you'd previously read through.


> I am not sure how feasible is to add content to the original td that contains the +/- icon and make it break out of the existing table structure, push the other rows down, and appear in the same way.

Yea I'm not sure it'd be easy with the way it is now, but I'm sure there's a `flex-flow` or `flex-direction` trick we can employ here.  Or maybe it'd be easier if expanding the `td` shifts everything to the right down so the new information shows on top of the row instead of on the bottom?

Reply:
I have removed the extra `tr` for the analytics data. I am using now a `td`.